### DISCUSSION TITLE
SVG to PNG

### DISCUSSION THREAD

**Discussion by dividerbeam** [chillerlan/php-qrcode#82] [Q&A]
In examples exist file example_svg.png with gradient from svg, how i can save svg like png? i can't find example of code

**Comment by codemasher** [comment]
This file is the image you can see in the readme. It's a screenshot of the SVG example.

**Reply by dividerbeam** [reply]
i fix it with simple Imagick:

$qrcode = (new QRCode($options))->render($data);


namespace Imagick;
use Imagick;
$im = new Imagick();
$im->readImageBlob($qrcode);

$im->setImageFormat("png");
$im->resizeImage(200, 200, Imagick::FILTER_LANCZOS, 1);

$im->writeImage('./'.$ckpp.'.png');
$im->clear();
$im->destroy();


but on server must be installed service Inkscape, without it gradient from svg not convert correctly to png
